### ISSUE TITLE
Don't report no-empty warnings when a parent is FunctionExpression / FunctionDeclaration

### ISSUE THREAD

**Issue by Constellation** [eslint/eslint#70]
When target is

``` js
function t() {
}
```

or

``` js
(function () {
}());
```

eslint should not report `no-empty` warnings.

version is 0.0.7-dev master.
